Tool complains about "mixed"

ComposerRequireChecker 3.1.0@b3cd6d30e2335997c5eabb19e3f01d699bc06caa

```
The following 1 unknown symbols were found:
+----------------+--------------------+
| Unknown Symbol | Guessed Dependency |
+----------------+--------------------+
| mixed          |                    |
+----------------+--------------------+
```
Error: Process completed with exit code 1.

--- a/data/config.dist.json
+++ b/data/config.dist.json
@@ -2,7 +2,7 @@
   "symbol-whitelist" : [
     "null", "true", "false",
     "static", "self", "parent",
-    "array", "string", "int", "float", "bool", "iterable", "callable", "void", "object"
+    "array", "string", "int", "float", "bool", "iterable", "callable", "void", "object", "mixed"
   ],
   "php-core-extensions" : [
     "Core",

--- a/src/ComposerRequireChecker/Cli/Options.php
+++ b/src/ComposerRequireChecker/Cli/Options.php
@@ -30,6 +30,7 @@ class Options
         'callable',
         'void',
         'object', // types
+        'mixed',
     ];
 
     /** @var array<string>  */

